src/formats/PDB.cpp

```diff
@@ -807,7 +808,8 @@ void PDBFormat::write_next(const Frame& frame) {
     }
 
     // Used to skip writing unnecessary connect records
-    auto is_atom_record = std::vector<bool>(frame.size(), false);
+    // use std::deque because std::vector<bool> is completly broken
```

Comment:
How so?

Reply:
std::vector<bool> is required by the standard to use bit packing, and this has non negligible performance issues: https://stackoverflow.com/a/36933356/4692076, https://stackoverflow.com/q/17794569/4692076.

"completly broken" might be a bit strong here, I'll rephrase it =)